src/include/daos_srv/vos_types.h

```diff
@@ -358,6 +358,8 @@ typedef struct {
 		struct {
 			/** Non-zero if punched */
 			daos_epoch_t		ie_punch;
+			/** If applicable, non-zero if object is punched */
+			daos_epoch_t		ie_obj_punch;
```

Comment:
I don't quite see the purpose of 'ie_obj_punch', It looks to me it is always equal to 'ie_punch' in this patch.

Reply:
I see the purpose now, thanks for the explanaiton.